src/scripts/main.js

```diff
@@ -1,3 +1,193 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+let score = 0;
+
+let board = [
+  [0, 0, 0, 0],
+  [0, 0, 0, 0],
+  [0, 0, 0, 0],
+  [0, 0, 0, 0],
+];
+
+const rows = 4;
+const columns = 4;
+
+const button = document.querySelector('button');
+const messageLose = document.querySelector('.message-lose');
+const messageWin = document.querySelector('.message-win');
+const messageStart = document.querySelector('.message-start');
+
+button.addEventListener('click', () => {
+  if (button.classList.contains('start')) {
+    button.classList.remove('start');
+    button.classList.add('restart');
+    button.innerText = 'Restart';
+
+    messageStart.classList.add('hidden');
+    setGame();
+    setGame();
```

Comment:
Do you need to call this function twice?

Reply:
> Do you need to call this function twice?

yes, because its generate one tile, but I need 2

Reply:
I`ve made some changes, so now -only once